src/Veldrid.VirtualReality/OpenVR/OpenVRContext.cs

```diff
@@ -31,6 +31,24 @@ internal class OpenVRContext : VRContext
 
         internal GraphicsDevice GraphicsDevice => _gd;
 
+        // The relative directories for the native binaries
+        private const string LibDir64 = "native/win-x64";
+        private const string LibDir32 = "native/win-x86";
+
+        // DllImport to load OpenVR native binary, might need to reimplement using NativeLibraryLoader, but given that we're loading a VR API, and most VR users use Windows, it may not be worth it
+        // Then again, this API is OpenVR, which is the only VR API other than OpenXR available on Linux, so it may be worth it
+        // TODO: Reimplement using NativeLibraryLoader
+        [System.Runtime.InteropServices.DllImport("kernel32.dll", CharSet = System.Runtime.InteropServices.CharSet.Auto, SetLastError = true)]
+        static extern bool SetDllDirectory(string lpPathName);
```

Comment:
I don't think any of this should be necessary in theory. The generated openvr_api.cs file uses `DllImport` to invoke the initialization functions. Those DllImport's should resolve to the proper 32/64-bit binary automatically, assuming those binaries are properly packaged and referenced via NuGet. The 64-bit binary is already deployed to the expected `runtimes/win-x64/native` path, and the same should work for the 32-bit binary.